tasks/setup-Debian.yml

```diff
@@ -31,7 +31,8 @@
     url: "{{ docker_apt_gpg_key }}"
     dest: /etc/apt/trusted.gpg.d/docker.asc
     mode: '0644'
-    force: true
+    force: false
+    checksum: sha256:1500c1f56fa9e26b9b8f42452a553675796ade0807cdce11975eb98170b3a570
```

Comment:
The issue I see with this approach is that the `checksum` value here would be associated with `docker_apt_gpg_key`, and for maintenance reasons, both would need changes over time (maybe) depending on how Docker maintains their repo. Also, this change could break existing installs for users who don't use the same apt repo.

I think checksum would need to be configurable, at minimum, and the value here could be the default.

Reply:
Thanks for the review and feedback.

> I think checksum would need to be configurable, at minimum, and the value here could be the default.

You mean something like:

```
    checksum: "{{ docker_apt_gpg_key_checksum | default(sha256:1500c1f56fa9e26b9b8f42452a553675796ade0807cdce11975eb98170b3a570) }}"
```

and the people would be able to overwrite it, if required in their environments?

Reply:
@geerlingguy Thanks again for your feedback -- any comment on the above?

Reply:
Yes, exactly!

Or just set `checksum: "{{ docker_apt_gpg_key_checksum | default(omit) }}`

and then in the defaults/main.yml vars file:

```
docker_apt_gpg_key_checksum: "sha256:1500c1f56fa9e26b9b8f42452a553675796ade0807cdce11975eb98170b3a570"
```